__tests__/e2e/docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+version: '3.7'
+services:
+  es01:
```

Comment:
Is there a reason to name these services with a number? I don't think we expect to add more in the future. Simply using `elasticsearch` is more readable, the same with kibana and synthetic